### PR TITLE
Hw7

### PR DESCRIPTION
**Pull request by Konstantin711**
Homework is finished

### CODE REVIEW

File: homework_7_task_2.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# and returns 3 values (using a tuple): the perimeter of the square,
+# the area of the square, and the diagonal of the square.
+import math
+
+
+def square(side: int) -> tuple:
+    """
+    Method to calculate square`s features
+    :param side: Side of the square
+    :return: Tuple: perimeter, area, diagonal
```
Comment (bohdandovbsyh) [inline homework_7_task_2.py:12]: you already have annotation. So it in no need in :paran and :return

File: homework_7_task_3.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# 3.
+# Write a function is_prime that takes 1 argument - a number from 2 to 1000,
+# and returns True if it is a prime number, and False otherwise.
+
+
+def is_prime(number: int) -> bool:
+    """
+    Method checks that digit is prime in range (2, 1000)
+    :param number: any int (2, 1000)
+    :return: bool
```
Comment (bohdandovbsyh) [inline homework_7_task_3.py:10]: same here

File: homework_7_task_3.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+if __name__ == '__main__':
+    for i in range(2, 1000):
+        if is_prime(i):
+            assert (str(i) in prime_to_1000), f'{i} exist in prime array'
```
Comment (bohdandovbsyh) [inline homework_7_task_3.py:36]: your function already returns True or False. Why do you need that big list at all?

Comment (Konstantin711) [reply homework_7_task_3.py:36]: I have used it for debugging of algorithm and decided to leave it. There are all prime digits till 1000 in it

File: homework_7_task_2.py (hunk 1 above)
Comment (Konstantin711) [reply homework_7_task_2.py:12]: Thanx. Kept it!

**Pull request merged**